make.py

```diff
@@ -78,52 +92,115 @@ def make_github_issue_no_notify(title, body, labels):
     pprint(response.json())
 
     if response.status_code == 202:
-        print('Successfully created Issue "%s"' % title)
+        print(f'Successfully created Issue "{title}"')
     else:
-        print('Could not create Issue "%s"' % title)
+        print(f'Could not create Issue "{title}"')
         print("Response:", response.content)
 
 
-template = Path("./issues/test_uncaught.markdown").read_text()
-
-for domain in [
-    # "abode",
-    # "cast",
-    # "config",
-    "deconz",
-    "default_config",
-    "demo",
-    "discovery",
-    "dsmr",
-    "dynalite",
-    "dyson",
-    "gdacs",
-    "geonetnz_quakes",
-    "homematicip_cloud",
-    "hue",
-    "ios",
-    "local_file",
-    "meteo_france",
-    "mikrotik",
-    "mqtt",
-    "plex",
-    "qwikswitch",
-    "rflink",
-    "samsungtv",
-    "tplink",
-    "tradfri",
-    "unifi_direct",
-    "upnp",
-    "vera",
-    "wunderground",
-    "yr",
-    "zha",
-    "zwave",
-]:
-    title = f"Fix {domain} tests that have uncaught exceptions"
-    body = template.replace("{{ DOMAIN }}", domain)
-    labels = [f"integration: {domain}", "to do"]
-    # print(title)
-    # print(body)
-    # print(labels)
-    make_github_issue(title=title, body=body, labels=labels)
+def common_issue_options(func):
```

Comment:
Why don't you use decorators ?

Reply:
I started with having multiple commands but then I refactored to just one command.

Reply:
Fixed.